tests/MiniCover.UnitTests/Commands/Options/MiniCoverTouchOptionTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Extensions.CommandLineUtils;
+using MiniCover.Commands.Options;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace MiniCover.UnitTests.Commands.Options
+{
+    public class MiniCoverTouchOptionTests
+    {
+        [Theory]
+        [MemberData(nameof(MiniCoverOptionsData))]
+        internal void TouchOption_DefaultValue_CreationTest(MiniCoverTouchOption option, string expected)
+        {
+            //arange
+            var cmd = new CommandLineApplication();
+            option.AddTo(cmd);
+
+            //act
+            option.Validate();
+            var actual = option.GetValue();
+            var fileInfo = new FileInfo(actual);
+
+            //assert
+            Assert.Equal(expected, fileInfo.FullName);
+            Assert.True(fileInfo.Exists);
+        }
+
+        public static IEnumerable<object[]> MiniCoverOptionsData
+        {
+            get
+            {
+                {
```

Comment:
There are extra brackets on all MiniCoverOptionsData.
But we can fix that later.